azuravian/TitleColorMatch.py

```diff
@@ -229,7 +243,16 @@ def _get_logo_color(self) -> tuple[str, str, int]:
 
             # First valid color, return color and stroke based on luminance
             luminance = (r * 0.299) + (g * 0.587) + (b * 0.114)
-            return hexcolor if luminance >= self.TITLE_MIN_LUMINANCE else self.TITLE_COLOR, 'black' if luminance < self.TITLE_MIN_LUMINANCE or luminance > 50 else 'white', luminance
+
+            # Determine returns
+            if luminance >= self.TITLE_MIN_LUMINANCE:
+                title_color = hexcolor
+                stroke_color = 'black' if luminance > 50 else 'white'
+            else:
+                title_color = self.TITLE_COLOR
+                stroke_color = 'black'
```

Comment:
I thought about this earlier and now I'm even more certain that there should probably be a STROKE_COLOR var thats used here

Reply:
What would the variable be for?